backend/api/events.py

```diff
@@ -33,20 +45,20 @@ def get_events(event_service: EventService = Depends()) -> list[EventDetails]:
 
 
 @api.get("/organization/{slug}", response_model=list[EventDetails], tags=["Events"])
-def get_events_by_organization(
+def get_events_from_organization(
     slug: str, event_service: EventService = Depends()
 ) -> list[EventDetails]:
     """
     Get all events from an organization
 
-    Parameters:
+    Args:
         slug: a valid str representing a unique Organization
         event_service: a valid EventService
 
     Returns:
         list[EventDetails]: All `EventDetails`s in the `Event` database table from a specific organization
     """
-    return event_service.get_events_by_organization(slug)
+    return event_service.get_events_from_organization(slug)

```

Comment:
Here's a place where I would probably refactor to two lines:

~~~
organization = organization_service.get_org_by_slug(slug)
return event_service.get_events_by_organization(organization)
~~~

It's potentially less efficient to do the extra query, however, I think it cleans up the `API` of event_service quite a bit. It's a leap to need to know the slug of an org in order to use the event service.

Reply:
There is another comment with more context later in the review on this, but I wonder if we should be passing in the user (or the lack of a user in a public-viewing case) and then returning models with a little more detail that include registration status.